README.md

```diff
@@ -344,7 +344,7 @@ Snapshots of the development version are available in [Sonatype's snapshots repo
 License
 -------
 
-    Copyright (C) 2017 Uber Technologies
+    Copyright (c) 2018 Uber Technologies
```

Comment:
This typically is the inception year. I am not sure we should be changing this. Please check with legal?

Reply:
Reverted for now 74859eb